packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lion/ui",
-  "version": "0.5.2",
+  "version": "0.5.2-alpha.0",
```

Comment:
Maybe 0.6.0-alpha.

Also, we need to do this to prevent npm from referring to it as latest: https://github.com/ing-bank/lion/commit/8702381b52cd9ded130318397e3e3e5b30b17a4d

(but then with lit3 as tag).